host/libcontainer_backend.go

```diff
@@ -448,7 +453,7 @@ func (l *LibcontainerBackend) Run(job *host.Job, runConfig *RunConfig, rateLimit
 		Cgroups: &configs.Cgroup{
 			Path: filepath.Join("/flynn", job.Partition, job.ID),
 			Resources: &configs.Resources{
-				AllowedDevices: configs.DefaultAllowedDevices,
+				AllowedDevices: append(configs.DefaultAllowedDevices, job.Config.AllowedDevices...),
```

Comment:
In order to allow restricting jobs further, it probably makes sense to use _only_ the provided devices and capabilities if specified so that the defaults can be overridden?